test/redis/store/ttl_test.rb

```diff
@@ -42,6 +43,21 @@ def setnx(*a)
     end
   end
 
+  def pipelined(&block)
+    @used_pipeline = true
+    instance_eval do
+      def setnx(*a)
+        @setnxes << a
+      end
+
+      block.call
+    end
+  end
+
+  def used_pipeline?
```

Comment:
given the comment below on the tests, I don't think you need this extra method anymore because you can use the test framework to tell you when `#pipelined` gets called.